ops/roles/cgus/tasks/database.yml

```diff
@@ -0,0 +1,72 @@
+- name: Define if {{ name }} base data has been obtained from the public instance
+  git:
+    repo: "{{ repository }}"
+    dest: "/home/{{ ansible_user }}/{{ directory }}"
+    clone: no
+    update: no
+    accept_hostkey: yes
+    key_file: "/home/{{ ansible_user }}/.ssh/cgus-bot-key"
+  register: existing_repository  # the `return` property of the return value can tell us if the repository has been cloned already or not, see <https://docs.ansible.com/ansible/latest/collections/ansible/builtin/git_module.html#return-values>
+  tags:
+    - restart
+    - start
+    - update
+
+- name: Remove existing locks in {{ name }}
+  file:
+    path: "/home/{{ ansible_user }}/{{ directory }}/.git/index.lock"
+    state: absent
+  when: existing_repository.before is defined and existing_repository.before  # if existing_repository.before is not null, then the repository already is not new
+  tags:
+    - restart
+    - start
+    - update
+
+- name: Obtain {{ name }} initial data from the public instance
+  git:
+    repo: "{{ repository }}"
+    dest: "/home/{{ ansible_user }}/{{ directory }}"
+    depth: 1
+    accept_hostkey: yes
+    key_file: "/home/{{ ansible_user }}/.ssh/cgus-bot-key"
+  when: existing_repository.before is defined and not existing_repository.before  # if existing_repository.before is null, then the repository already is new
+  tags:
+    - setup
+
+- name: Get latest data from public instance of {{ name }}
+  command:
+    cmd: git pull origin master:master
```

Comment:
If there are unpushed commits, we get this error:
```
    From github.com:ambanum/CGUs-versions
     ! [rejected]        master     -> master  (non-fast-forward)
```
It works if we replace `git pull origin master:master` by `git pull`